azkaban-web-server/build.gradle

```diff
@@ -50,27 +71,55 @@ sourceSets {
   }
 }
 
-bower {
-  installBase = 'bower' // <1>
+task movingJsTojsToPackage(dependsOn: ['npm_install']) {
 
-  'moment'('2.14.1'){
-    source 'min/moment.min.js' >> '/'
+  doLast {
+    copy {
+      from "node_modules/later/later.min.js"
+      into 'build/jsToPackage'
+    }
+    copy {
```

Comment:
Yes. I considered moving copyJS to Distribution Task. I was worried that would make Distribution Task looks Ugly.